info.json

```diff
@@ -0,0 +1,192 @@
+    "version": "1.0.2",
+    "label": "GCP CA Service",
+    "description": "This connector integrates with GCP CA Service .",
+    "publisher": "Fortinet_CSE",
+    "icon_small_name": "cloudsec-png-small.png",
+    "icon_large_name": "cloudsec-png-large.png",
+    "category": [
+        "Utilities"
+    ],
+    "configuration": {
+        "fields": [
+            {
+                "title": "Upload Service Account JSON File",
+                "type": "file",
+                "name": "auth_file",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "tooltip": "The service account private key file."
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "list_certificate_authorities",
+            "title": "get_ca_crl",
+            "description": "get CA cert and CRL",
+            "enabled": true,
+            "category": "investigation",
+            "annotation": "get_ca_cert_crl",
+            "parameters": [
+                {
+                    "title": "location",
+                    "type": "text",
+                    "name": "location",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "tooltip": "GCP region for CA Service"
+                },
+                {
+                    "title": "ca_pool_name",
+                    "type": "text",
+                    "name": "ca_pool_name",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "tooltip": "GCP CA Service Pool Name."
+                }
+            ],
+            "output_schema": {
+                "kind": "",
+                "id": "",
+                "items": [],
+                "nextPageToken": "",
+                "selfLink": "",
+                "warning": {}
+            },
+            "open": false
+        },
+        {
+            "operation": "csr",
+            "title": "csr",
+            "description": "submit certifcate signing request",
+            "enabled": true,
+            "category": "investigation",
+            "annotation": "csr",
+            "parameters": [
+                {
+                    "title": "location",
+                    "type": "text",
+                    "name": "location",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "tooltip": "GCP region for CA Service"
+                },
+                {
+                    "title": "ca_pool_name",
+                    "type": "text",
+                    "name": "ca_pool_name",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "tooltip": "GCP CA Service Pool Name."
+                },
+                {
```

Comment:
Action parameter title should be in title case; it should be 'CA Name'.

It's applicable for all actions -- > all action parameter titles should be in title case